app/views/settings/index.html.erb

```diff
@@ -3,9 +3,9 @@
 <h1 class="govuk-heading-l">Settings</h1>
 
 <%= render "account" %>
-
+<br>
 <%= render "organisation" %>
-
+<br>
```

Comment:
The prototype uses CSS to control the spacing (I see `govuk-!-margin-bottom-9"` used on the `dl` elements, which should be possible to use on the tables as well).

Reply:
Thank you!